<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QDialog" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Вагон</string>
   </property>
   <widget class="QPushButton" name="carAdd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carRem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="carID">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="carNSeats">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Мест:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="carView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>241</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Место</string>
   </property>
   <widget class="QLineEdit" name="seatCarID">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Вагон:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="seatNum">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="seatRem">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Возврат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seatAdd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Продать</string>
    </property>
   </widget>
   <widget class="QTableView" name="seatView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
